playbooks/roles/deploy-osh/files/020-ingress.sh

```diff
@@ -54,8 +54,8 @@ EOF
   helm upgrade --install ingress-${NAMESPACE} ${OSH_INFRA_PATH}/ingress \
     --namespace=${NAMESPACE} \
     --values=/tmp/ingress-${NAMESPACE}.yaml \
-  ${OSH_EXTRA_HELM_ARGS} \
-  ${!OSH_EXTRA_HELM_ARGS_INGRESS_NAMESPACE}
+  "${OSH_EXTRA_HELM_ARGS}" \
```

Comment:
This seems to generate invalid input for helm CLI.

Reply:
Same applies for horizon change.

Reply:
done, actually I disabled the checking for hte file as it comes from upstream, we should not modify for aesthetic reasons.